CONTRIBUTING.rst

```diff
@@ -11,6 +11,58 @@ Features and Bugs
 The easiest way to contribute is to fork this repository and submit a pull request.
 You can also submit an issue or write an email to us at support@strateos.com if you want to discuss ideas or bugs.
 
+Dev Env Setup
+-------------
+
+Installation:
+    Minimum Python version supported for development is :code:`v3.6`.
+
+Python Virtual Environment (Optional):
+    Use of virtual environment to isolate development environment is
+    highly recommended. There are several tools available such as
+    conda_ and pyenv_.
+
+Dependencies:
+    We recommend first activating your desired virtualenv, then
+    installing the dependencies using the snippet below.
+
+.. code-block:: bash
```

Comment:
extremely minor nit: these aren't really using anything bash-specific and could just be `sh` which *appears* to be a supported tag

Reply:
cool! will do

Reply:
yup sh works